curriculum/challenges/portuguese/02-javascript-algorithms-and-data-structures/basic-javascript/comparison-with-the-greater-than-operator.md

```diff
@@ -22,7 +22,7 @@ Tal como o operador de igualdade, o operador maior que converterá os tipos de d
 '1' >  9
 ```
 
-Em ordem, essas expressões seriam iguais à `true`, `false`, `true` e `true`.
+Em ordem, essas expressões seriam iguais à true, `true`, `true`, `false` e `false`.
```

Comment:
Does this not still look incorrect?

Reply:
Yes I would expect it to be
```suggestion
Em ordem, essas expressões seriam iguais à `true`, `true`, `false` e `false`.
```

now the values in backticks match, but it seems there is an extra one before outside of backticks

Reply:
Yep that's incorrect.

Reply:
@ShaunSHamilton I approved your fix over on Crowdin.

Reply:
left a message on the Portuguese chat room, I don't know if we have Portuguese proofreaders on GitHub to tag as needed

Reply:
Or that was faster

Reply:
@nhcarrigan Do we merge this anyway? Or, will Dr. Camperbot rebase with an update?

Reply:
Camperbot will rebase. I'll go ahead and re-run the action :)